Formula/planck.rb

```diff
@@ -4,6 +4,7 @@ class Planck < Formula
   url "https://github.com/planck-repl/planck/archive/2.27.0.tar.gz"
   sha256 "d69be456efd999a8ace0f8df5ea017d4020b6bd806602d94024461f1ac36fe41"
   license "EPL-1.0"
+  revision 1
```

Comment:
If we need another run, will defer this to later as it will probably fail on Linux given `webkitgtk` dependency
```suggestion
```